src/app/cluster/cluster-details/cluster-secrets/component.ts

```diff
@@ -55,6 +55,14 @@ export class ClusterSecretsComponent implements OnInit {
           return this.getIconClass(this.health.gatekeeperAudit);
         case 'gatekeeperController':
           return this.getIconClass(this.health.gatekeeperController);
+        case 'mlaGateway':
```

Comment:
There is a lot of components already. I know this may be a bit out of scope but can you add enum for these values and use it globally?

Reply:
Sure, will add :)

Reply:
Thanks.